debian/copyright

```diff
@@ -0,0 +1,26 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cometvisu
+Source: https://github.com/CometVisu/CometVisu
+
+Files: *
+Copyright: Christian Mayer <CometVisu@ChristianMayer.de>
+           Tobias Bräutigam <tbraeutigam@gmail.com>
+           Michael Markstaller <devel@wiregate.de> / Elaborated Networks GmbH
```

Comment:
I just went through some file headers and copied the copyright notices from there. The actual syntax should have years, and then it makes perfect sense again to leave it as is (for a period 201x - 2014?) and add the new address for the period afterwards.